src/main/resources/org/biouno/unochoice/stapler/unochoice/unochoice.js

```diff
@@ -1070,7 +1070,7 @@ var UnoChoice = UnoChoice || (function($) {
         var value = '';
         if (e.attr('name') === 'value') {
             value = getElementValue(htmlParameter);
-        }  else if (e.prop('tagName') === 'DIV') {
+        }  else if (e.prop('tagName') === 'DIV' || e.prop('tagName') === 'SPAN') {
```

Comment:
This is the other part of the fix. Where the value is retrieved when you change the value in the UI.